src/main/java/com/synopsys/integration/alert/web/documentation/SwaggerConfiguration.java

```diff
@@ -41,14 +41,14 @@
 @EnableSwagger2
 public class SwaggerConfiguration {
     public static final String SWAGGER_DEFAULT_URL = "/swagger-ui.html";
-    public static final String[] SWAGGER_WEB_RESOURCE_DIRECTORIES = new String[] {
-        SWAGGER_DEFAULT_URL,
-        "/webjars",
-        "/webjars/**",
-        "/swagger-resources",
-        "/swagger-resources/**",
-        "/v2/api-docs"
-    };
+    //public static final String[] SWAGGER_WEB_RESOURCE_DIRECTORIES = new String[] {
+    //        SWAGGER_DEFAULT_URL,
+    //        "/webjars",
+    //        "/webjars/**",
+    //        "/swagger-resources",
+    //        "/swagger-resources/**",
+    //        "/v2/api-docs"
+    //    };
```

Comment:
Can we add a comment here as to why this is comented out?

Reply:
Alternatively I can delete this. Having public mutable fields is being reported as a vulnerability and this field was not even being referenced anywhere. So I commented it out to remove the vulnerability until we determine what the purpose of this String array was.

Reply:
I would delete it. If we need it back we can just get it from the files history.